cost/aws/old_snapshots/aws_delete_old_snapshots.pt

```diff
@@ -443,31 +437,32 @@ script "js_snapshots_cost_mapping", type: "javascript" do
       var monthly_savings = sum*30;
       total=total+monthly_savings;
       result.push({
-        accountId: ds_get_caller_identity[0]['account'],
+        accountID: ds_get_caller_identity[0]['account'],
         snapshotId : snapshot['snapshotId'],
         region : snapshot['region'],
-        tagKeyValue : snapshot['tagKeyValue'],
+        tags : snapshot['tagKeyValue'],
         daysOld : snapshot['daysOld'],
         imageId : snapshot['imageId'],
         snapshotImageIdFormat : snapshot['snapshotImageIdFormat'],
-        savings : cur + ' '+formatNumber(monthly_savings.toFixed(3), separator),
+        savings : Number(monthly_savings.toFixed(3)),
```

Comment:
Ah, looks like the `Number` function isn't available: the savings in the incident are now all showing up as zero.

Maybe we can just remove `toFixed` altogether and use the raw number?
```suggestion
        savings : monthly_savings,
```

Reply:
Or `parseFloat` would probably work as an alternative to `Number`, at least I've seen that function called in these policies.e